word_guess.rb

```diff
@@ -0,0 +1,213 @@
+# syntax: print pie array with index[1]
+
+#def froggy
+#end
+
+# #
+#      @ . . @
+#     (-------)
+#   (   >___<   )
+#   ^^  ~   ~   ^^
+#   \\\\\\\///////
+# #
+# #   @ . . @
+#  (-------)
+# (   >___<   )
+#
+#
+#
+#
+# ^^  ~   ~   ^^
+# \\\\\\\///////
+#
+#     @ . . @
+#    (-------)
+#   (   >___<   )
+#  ^^  ~   ~   ^^
+#  \\\\\\\///////
+
+#Reprint the ASCII image, the guessed letters the status of the mystery word
+# (create a method)
+
+#When word mystery word is complete
+# puts the correct word
+# print ASCII art
+# puts "you won!"
+# exit
+
+#When ASCII art is gone!
+# puts "you lost"
+# exit
+
+
+class WordGuess
+
+   frog1 = "       @ . . @
+      (-------)
+    (   >___<   )
+    ^^  ~   ~   ^^ ".colorize(:green) +
+    "\n   //////\\\\\\\///////
+    ~~~~~~~~~~~~~~~~
+    ~~~~~~~~~~~~~~~~ ".colorize(:blue)
+
+  frog2 = "       @ . . @
+      (-------)
+    (   >___<   )
+    ^^  ~   ~   ^^ ".colorize(:green) +
+      "\n   ~~~~~~~~~~~~~~
+      ~~~~~~~~~~~~~~
+      ~~~~~~~~~~~~~~ ".colorize(:blue)
+
+  frog3 = "       @ . . @
+      (-------)
+    (   >___<   ) ".colorize(:green) +
+    "\n   ~~~~~~~~~~~~~~
+    ~~~~~~~~~~~~~~
+    ~~~~~~~~~~~~~~
+    ~~~~~~~~~~~~~~".colorize(:blue)
+
+  frog4 = "       @ . . @
+      (-------)".colorize(:green) +
+      "\n   ~~~~~~~~~~~~~~
+      ~~~~~~~~~~~~~~
+      ~~~~~~~~~~~~~~
+      ~~~~~~~~~~~~~~
+      ~~~~~~~~~~~~~~ ".colorize(:blue)
+
+  frog5 = "       @ . . @".colorize(:green) +
+  "\n   ~~~~~~~~~~~~~~
+  ~~~~~~~~~~~~~~
+  ~~~~~~~~~~~~~~
+  ~~~~~~~~~~~~~~
+  ~~~~~~~~~~~~~~ ".colorize(:blue)
+
+  frog6 =  "  ~~~~~~~~~~~~~~
+  ~~~~~~~~~~~~~~
+  ~~~~~~~~~~~~~~
+  ~~~~~~~~~~~~~~
+  ~~~~~~~~~~~~~~
+  ~~~~~~~~~~~~~~ ".colorize(:blue)
+
+  frogarray = []
+  frogarray.push(frog1, frog2, frog3, frog4, frog5, frog6)
+#Create an array of possible words for the player to guess. Use sample to get one word from array.
+mystery_words = %w(get red kitty nose)
+
+  puts "Welcome to Word Guess.\nYour goal is to guess the correct word.\nEach time you guess a wrong letter not in the mystery word, the frog will sink into the water a little more."
+  puts "You must guess the correct word before the frog is submerged."
+  puts "Let's get started!"
+  #Print ASCII art and mystery word.
+  puts frogarray[0]
+  # guesses = false
+  # until guesses == words_display ||
+  # sample is a method on the array class
+  # chars is a method on the string class and returns an array of characters
+  words_display = mystery_words.sample.chars
+  # print words_display
+  # replace the letter in mystery word with dashes
+  n = words_display.length
+  dashed_word = ("-" * n).chars.to_a
+  p dashed_word
+
+
+  # index where they have guessed
+  #guessed_letter = []
+  wrong_guesses = []
+  # display for each turn
+  # store the wrong guess in an array
+  while dashed_word.include?("-") && wrong_guesses.length < 5
+    puts "What letter do you want to guess?"
+    guessed_letter = gets.chomp
```

Comment:
It would be good to put this line and the `puts` above within a loop to ensure the user is entering a valid letter.